src/view3d.cpp

```diff
@@ -355,18 +518,18 @@ void View3D::drawMeshes(const TransformState& transState,
         meshFaceShader.setUniformValue("lightDir_eye", lightDir.x, lightDir.y, lightDir.z);
         for (size_t i = 0; i < geoms.size(); ++i)
             geoms[i]->drawFaces(meshFaceShader, transState);
-        meshFaceShader.release();
+        // meshFaceShader.release();
```

Comment:
I wanted to highlight that we should not "release" the shader, which is equivalent to calling `glUseProgram(0)`, at least that's what the Qt specs say. Should we keep that information in a better written comment or should we rather remove it everywhere?